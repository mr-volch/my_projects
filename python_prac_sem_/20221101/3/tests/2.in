f, g, h = Grange(3, 3, 729), Grange(3, 9, 2187), Grange(9, 9, 2187)
o, e = Grange(1, 1, 1), Grange(27, 3, 9)
print(f"{f=}, {o=}, {h=}")
print(f"{list(o)}({not(not(o))}), {list(g)}({not(g)})")
print(f"{len(f)=}, {len(o)=}, {len(e)=}")
print(f"{len(list(f))=}, {len(tuple(g))=}, {len(list(o))=}")
print(f"{f[421]=}, {h[244]=}")
a, b = f[3:532525], h[3:53252:3]
print(f"{a=}, {a[1]=}")
print(f"{b=}, {b[1]=}, {tuple(b)=}") 
