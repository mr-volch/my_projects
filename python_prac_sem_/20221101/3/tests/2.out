f=grange(3, 3, 729), o=grange(1, 1, 1), h=grange(9, 9, 2187)
[](False), [3, 27, 243](False)
len(f)=5, len(o)=0, len(e)=0
len(list(f))=5, len(tuple(g))=3, len(list(o))=0
f[421]=2213958580549235792540033700607174209967503636621247224593767164520597787597881701074737106687986558831280592677708220474936674312204961945819562966562110151186557893429570252320571762658691821032651609, h[244]=615764734303204742542384580256114737218638504055981016449830409195424142518652229236710329776615840798458563270070684613674337101678488478094335736702772710461172437261387042119686071509522823586726115117786119641789164693561953553449
a=grange(3, 3, 532525), a[1]=9
b=grange(3, 729, 53252), b[1]=2187, tuple(b)=(3, 2187)
